Formula/rure.rb

```diff
@@ -0,0 +1,32 @@
+class Rure < Formula
+  desc "C API for RUst's REgex engine"
+  homepage "https://github.com/rust-lang/regex/tree/master/regex-capi"
```

Comment:
```suggestion
  homepage "https://github.com/rust-lang/regex/tree/HEAD/regex-capi"
```
Let's not hardcode the branch name.